pontoon/base/models.py

```diff
@@ -100,8 +100,18 @@ def user_translated_locales(self):
 
 
 @property
+def user_name_or_email(self):
```

Comment:
Somehow, i feel we should use more generic for the methods, because they'll be easier to change in future.